packages/rum-core/src/domain/rumEventsCollection/view/viewMetrics/trackFirstInputTimings.spec.ts

```diff
@@ -1,23 +1,58 @@
-import { noop, type Duration, type RelativeTime } from '@datadog/browser-core'
+import {
+  noop,
+  type Duration,
+  type RelativeTime,
+  resetExperimentalFeatures,
+  ExperimentalFeature,
+  addExperimentalFeatures,
+} from '@datadog/browser-core'
 import { restorePageVisibility, setPageVisibility } from '@datadog/browser-core/test'
+import type { RumFirstInputTiming } from '../../../../browser/performanceCollection'
 import type { TestSetupBuilder } from '../../../../../test'
 import { setup } from '../../../../../test'
+import type { LifeCycle } from '../../../lifeCycle'
 import { LifeCycleEventType } from '../../../lifeCycle'
 import type { RumConfiguration } from '../../../configuration'
-import { FAKE_FIRST_INPUT_ENTRY } from '../setupViewTest.specHelper'
 import { resetFirstHidden } from './trackFirstHidden'
 import { trackFirstInputTimings } from './trackFirstInputTimings'
 
 describe('firstInputTimings', () => {
   let setupBuilder: TestSetupBuilder
   let fitCallback: jasmine.Spy<
-    ({ firstInputDelay, firstInputTime }: { firstInputDelay: number; firstInputTime: number }) => void
+    ({
+      firstInputDelay,
+      firstInputTime,
+      firstInputTargetSelector,
+    }: {
+      firstInputDelay: number
+      firstInputTime: number
+      firstInputTargetSelector?: string
+    }) => void
   >
   let configuration: RumConfiguration
+  let target: HTMLButtonElement
+
+  function newFirstInput(lifeCycle: LifeCycle, overrides?: Partial<RumFirstInputTiming>) {
+    lifeCycle.notify(LifeCycleEventType.PERFORMANCE_ENTRIES_COLLECTED, [
+      {
+        entryType: 'first-input',
+        processingStart: 1100 as RelativeTime,
+        startTime: 1000 as RelativeTime,
+        duration: 0 as Duration,
+        target,
+        ...overrides,
+      },
+    ])
+  }
 
   beforeEach(() => {
     configuration = {} as RumConfiguration
     fitCallback = jasmine.createSpy()
+
+    target = document.createElement('button')
+    target.setAttribute('id', 'fid-target-element')
+    document.body.appendChild(target)
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏What about introducing an `appendElement(tagName)` and use registerCleanupTask to remove it automatically?

Reply:
Yes, I'll extends its usage to all tests in a following PR